prog/storage/remove_spdk.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class Prog::Storage::RemoveSpdk < Prog::Base
+  subject_is :spdk_installation
+
+  def self.assemble(spdk_installation_id)
```

Comment:
How is this intended to be used? It has no deadlines and no supervising strand. Is it console invoked, with the operator Being Careful? (we have a few others like this, such as InstallRhizome, though to do is is starting to be less than the standard of adequacy to which we should aspire)

@furkansahin also added another prog kinda like this and I gave similar mumblings. Just copying in him because we might want to do something like have a special repl/console/operator supervision strand that these end up in by default, decide that deadlines are the way to get supervision, or both, or something like that.

Reply:
Yes, this is supposed to be manually by the operator, similar to InstallRhizome. I updated the commit message.